druid/ingestion-spec/ml_survey_evidence_kafka.json

```diff
@@ -0,0 +1,105 @@
+{
+  "type": "kafka",
+  "dataSchema": {
+    "dataSource": "sl-survey-evidence",
+    "parser": {
+      "type": "string",
+      "parseSpec": {
+        "format": "json",
+        "timestampSpec": {
+          "column": "completedDate",
+          "format": "auto"
+        },
+        "dimensionsSpec": {
+          "dimensions": [
+            {
+              "type": "string",
+              "name": "total_evidences"
+            },
+            {
+              "type": "string",
+              "name": "surveySubmissionId"
+            },
+            {
+              "type": "string",
+              "name": "createdBy"
+            },
+            {
+              "type": "string",
+              "name": "solutionExternalId"
+            },
+            {
+              "type": "string",
+              "name": "solutionId"
+            },
+            {
+              "type": "string",
+              "name": "surveyId"
+            },
+            {
+              "type": "string",
+              "name": "remarks"
+            },
+            {
+              "type": "string",
+              "name": "questionId"
+            },
+            {
+              "type": "string",
+              "name": "questionExternalId"
+            },
+            {
+              "type": "string",
+              "name": "questionResponseType"
+            },
+            {
+              "type": "string",
+              "name": "questionName"
+            },
+            {
+              "type": "string",
+              "name": "evidence_count"
+            },
+            {
+              "type": "string",
+              "name": "fileName"
+            },
+            {
+              "type": "string",
+              "name": "fileSourcePath"
+            },
+            {
+              "type": "string",
+              "name": "completedDate"
+            },
+            {
+              "type": "string",
+              "name": "appName"
+            }
+          ]
+        }
+      }
+    },
+    "metricsSpec": [],
+    "granularitySpec": {
+      "type": "uniform",
+      "segmentGranularity": "DAY",
+      "queryGranularity": "none",
+      "rollup": false
+    }
+  },
+  "tuningConfig": {
+    "type": "kafka",
+    "reportParseExceptions": false
+  },
+  "ioConfig": {
+    "topic": "<env>.ml.survey.evidence.druid",
+    "consumerProperties": {
+      "bootstrap.servers": "broker_ip:broker_port"
+    },
+    "taskCount": "<ml_survey_evidence_taskCount>",
```

Comment:
Fix these fields also

Reply:
can you please share me the sample ingestion spec which is correct ? , because i go throughed the already existing ingestion spec of diksha.

Reply:
Resolved, Please check